src/app/technical-record/weights/weights.component.html

```diff
@@ -3,10 +3,13 @@
   <tr class="govuk-table__row">
     <th scope="col" class="govuk-table__header"></th>
     <th scope="col" class="govuk-table__header">GB weight (kg)</th>
-    <th scope="col" class="govuk-table__header">EEC weight (kg)</th>
+    <th scope="col" class="govuk-table__header" *ngIf="activeRecord.vehicleType === 'hgv' || activeRecord.vehicleType === 'trl'">EEC weight (kg)</th>
```

Comment:
let evaluate this logic `activeRecord.vehicleType === 'hgv' || activeRecord.vehicleType === 'trl'"`. Anticipating complexity